benchmarks/src/main/java/org/elasticsearch/benchmark/routing/allocation/AllocationBenchmark.java

```diff
@@ -146,7 +146,7 @@ public void setUp() throws Exception {
             .build();
     }
 
-    private int toInt(String v) {
+    int toInt(String v) {
```

Comment:
Could you revert these benchmark-related changes too?

Reply:
done